react/components/EXPERIMENTAL_Table/SimpleTable/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC, memo } from 'react'
+
+import Cell from './Cell'
+import useTableContext from '../hooks/useTableContext'
+import { constants } from '../util'
+
+const SimpleTable: FC = () => {
+  const { columns, items } = useTableContext()
+
+  const renderHeadingRow = (headerData: string, headingIndex: number) => {
+    const headerRender = columns[headerData].headerRender
+    const content = headerRender
+      ? headerRender({ headerData })
+      : columns[headerData].title
+    return <Cell key={`col-${headingIndex}`} content={content} isHeading />
+  }
+
+  const renderRow = (rowData: Object, rowIndex: number) => {
+    return (
+      <tr
+        style={{ height: constants.ROW_HEIGHT }}
+        className="w-100 h-100 ph4 truncate"
+        key={`row-${rowIndex}`}>
+        {Object.keys(rowData).map((cel: string, cellIndex: number) => {
+          const cellRender = columns[cel].cellRender
+          const cellData = rowData[cel]
+          const content = cellRender
+            ? cellRender({ cellData, rowData })
+            : cellData
+          return <Cell key={`${rowIndex}-${cellIndex}`} content={content} />
+        })}
+      </tr>
+    )
+  }
+
+  const headings = (
```

Comment:
why `headings` with an s? Any chance there can be multiple? (just a semantic question).

Reply:
changed to `header` :)